fe/fe-core/src/main/java/org/apache/doris/common/proc/LoadProcDir.java

```diff
@@ -57,18 +57,26 @@ public LoadProcDir(Load load, Database db) {
 
     @Override
     public ProcResult fetchResult() throws AnalysisException {
-        Preconditions.checkNotNull(db);
         Preconditions.checkNotNull(load);
 
         BaseProcResult result = new BaseProcResult();
         result.setNames(TITLE_NAMES);
 
         // merge load job from load and loadManager
-        LinkedList<List<Comparable>> loadJobInfos = load.getLoadJobInfosByDb(db.getId(), db.getFullName(),
+        LinkedList<List<Comparable>> loadJobInfos;
+
+        // db is null means need total result of all databases
+        if (db == null) {
+            loadJobInfos = load.getAllLoadJobInfos();
```

Comment:
I think we can refactor this class this time, to remove `load` instance, only use `LoadManager`.
Because there is no more load job saved in `load` class. It should be deprecated in future.

Reply:
fix